<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestaurantInfoForm</class>
 <widget class="QWidget" name="RestaurantInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>95</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_RestaurantName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Restaurant name</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icons/Resources/close.png</normaloff>:/Icons/Resources/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="_label_price">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">TextLabel</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShow">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>60</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>60</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listComment">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>260</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtComment">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>180</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCommit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
